src/placeos-core/module_manager.cr

```diff
@@ -151,6 +155,20 @@ module PlaceOS
       Log.info { {message: "started module", module_id: mod.id, name: mod.name, custom_name: mod.custom_name} }
     end
 
+    def restart_module(mod : Model::Module)
+      mod_id = mod.id.as(String)
+      manager = proc_manager_by_module?(mod_id)
+
+      if manager
+        Log.info { {message: "restarting module", module_id: mod.id, name: mod.name, custom_name: mod.custom_name} }
+        manager.stop(mod_id)
+        start_module(mod)
+        Log.info { {message: "restarted module", module_id: mod.id, name: mod.name, custom_name: mod.custom_name} }
```

Comment:
I'm thinking remove this `restarted` line as we've already got the log entry generated by `start_module`